parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -407,11 +416,11 @@ public TypeMapping convertINT32(PrimitiveTypeName primitiveTypeName) throws Runt
           case DATE:
             return field(new ArrowType.Date(DateUnit.DAY));
           case TIMESTAMP_MICROS:
-            return field(new ArrowType.Timestamp(org.apache.arrow.vector.types.TimeUnit.MICROSECOND, "UTC"));
+            return field(new ArrowType.Timestamp(TimeUnit.MICROSECOND, "UTC"));
           case TIMESTAMP_MILLIS:
-            return field(new ArrowType.Timestamp(org.apache.arrow.vector.types.TimeUnit.MILLISECOND, "UTC"));
+            return field(new ArrowType.Timestamp(TimeUnit.MILLISECOND, "UTC"));
           case TIME_MILLIS:
-            return field(new ArrowType.Time(org.apache.arrow.vector.types.TimeUnit.MILLISECOND, 32));
+            return field(new ArrowType.Time(TimeUnit.MILLISECOND, 32));
           default:
           case TIME_MICROS:
```

Comment:
Shouldn't `TIME_MICROS` be supported as well?

Reply:
@rdblue Thanks. That's true. And this have to add some tests about converting from parquet to arrow. I will fix.